### PR TITLE
Update status-color so it respects theme colors for text-success

### PR DESCRIPTION
**Pull request by chrisdrackett**
Right now if a theme changes @text-color-success this package will not use the value from the theme. This update makes sure that the color set in the theme will be used.

### CODE REVIEW

**Comment by robwise** [comment]
@chrisdrackett Wasn't this already addressed in https://github.com/prettier/prettier-atom/pull/400?

**Comment by chrisdrackett** [comment]
@robwise in my testing (and I'm no expert, so maybe I'm missing something) in my theme if I set `@text-color-success` to another value it will not get picked up by prettier and it will still use the default color. However if I use this branch my theme is able to adjust the text color. I'm guessing that the prettier code is evaluated before my theme css can come in and overwrite?

File: styles/statusTile.less [hunk 1]
```diff
@@ -12,7 +12,7 @@ div {
 
   // If Format on Save is enabled, make the label more visible.
   &[data-prettier-format-on-save='enabled'] {
-    color: @text-color-success;
+    // color is handled by setting `text-success`
```
Comment (robwise) [inline styles/statusTile.less]: Let's change this to say "// color is handled by adding or removing the `text-success` class when updating the status tile"

**Comment by chrisdrackett** [comment]
@robwise things should be good now!

**Comment by robwise** [comment]
@chrisdrackett Can you make the commit messages conform to the conventional changelog angular standard? Otherwise they won't be picked up in the changelog

**Comment by chrisdrackett** [comment]
@robwise I'm happy to help out but I'm not sure what that is and don't have a lot of free time right now to figure it out

**Comment by chrisdrackett** [comment]
@robwise ok, took a crack, hopefully this works

**Comment by robwise** [comment]
@chrisdrackett Fixed for you in #435, still had some old commits in this PR that were not properly named

**Pull request closed without merging**